doc/manual/modules.rst

```diff
@@ -290,12 +298,20 @@ initialization steps that must occur at *runtime* from steps that can
 occur at *compile time*.  For this purpose, Julia allows you to define
 an ``__init__()`` function in your module that executes any
 initialization steps that must occur at runtime.
+This function will not be called during compilation
+(``--output-*`` or ``__precompile__()``).
+You may, of course, call it manually if necessary,
+but the default is to assume this function deals with local state
+which does not need to be captured in the compiled image.
+And it will be called anytime the module is loaded,
```

Comment:
In modern usage, "anytime" is considered rather informal. 

* https://books.google.com/ngrams/graph?content=anytime%2Cany+time&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Canytime%3B%2Cc0%3B.t1%3B%2Cany%20time%3B%2Cc0
*  https://www.grammarly.com/handbook/grammar/adjectives-and-adverbs/14/anytime-vs-any-time/

When you have a choice between a spelling that will look "sloppy" to a large fraction of readers and a spelling that is universally considered correct, it's best to choose the latter unless you are actively trying for an informal feel.